### PR TITLE
[IE TEMPLATE] Fix operation set in QueryNetwork

### PR DESCRIPTION
**Pull request by iefode**
### Tickets:
 - *72066*

### CODE REVIEW

File: docs/template_plugin/src/template_plugin.cpp [hunk 1]
```diff
@@ -152,14 +152,26 @@ InferenceEngine::QueryNetworkResult Plugin::QueryNetwork(const InferenceEngine::
...
-    for (auto&& node : transformedFunction->get_ops()) {
+    std::vector<ngraph::OpSet> opsets{ ngraph::get_opset1(),
+                                       ngraph::get_opset2(),
+                                       ngraph::get_opset3(),
+                                       ngraph::get_opset4(),
+                                       ngraph::get_opset5(),
+                                       ngraph::get_opset6(),
+                                       ngraph::get_opset7(),
+                                       ngraph::get_opset8() };
+    std::set<ngraph::NodeTypeInfo> op_super_set;
```
Comment (ilya-lavrenov) [inline docs/template_plugin/src/template_plugin.cpp:163]: why not `ngraph::OpSet` instead of `std::set<ngraph::NodeTypeInfo>`?

Comment (iefode) [reply docs/template_plugin/src/template_plugin.cpp]: Fixed, please review again)

**Review by ilya-lavrenov — APPROVED** [review]

File: docs/template_plugin/src/template_plugin.cpp [hunk 2]
```diff
@@ -152,14 +152,24 @@ InferenceEngine::QueryNetworkResult Plugin::QueryNetwork(const InferenceEngine::
     // So we need store as supported either unsupported node sets
     std::unordered_set<std::string> supported;
     std::unordered_set<std::string> unsupported;
-    auto opset = ngraph::get_opset4();
+    ngraph::OpSet op_super_set;
+#define _OPENVINO_OP_REG(NAME, NAMESPACE) op_super_set.insert<NAMESPACE::NAME>();
```
Comment (ilyachur) [inline docs/template_plugin/src/template_plugin.cpp:156]: Can we use `get_opsetN()`?
In other case we cannot use grep to find all places with opsets in order to add new opset.

Comment (iefode) [reply docs/template_plugin/src/template_plugin.cpp:156]: We should get all operations from all opsets :) The comment from @ilya-lavrenov was about previous technical decision. The details:
image

Comment (iefode) [reply docs/template_plugin/src/template_plugin.cpp:156]: @ilyachur ?

**Review by ilyachur — APPROVED** [review]

**Pull request merged**